docs/tldr/pages/linux/tmt-run.md

```diff
@@ -0,0 +1,36 @@
+# tmt run
+
+> `run` command is used to execute test steps. By default all test steps are run. See also: `tmt`, `tmt-try`.
+> More information: <https://tmt.readthedocs.io/en/stable/overview.html#run>.
+
+- Run all test steps for each plan:
+
+`tmt run`
+
+- Run selected plans and tests:
+
+`tmt run plan -n <plan_name> test -n <test_name>`
+
+- Show what tests a specific plan would run:
+
+`tmt run -vv discover plan -n <plan_name>`
+
+- Show test results from the last run in a web browser:
+
+`tmt run -l report --how html --open`
+
+- Add context to a test run:
+
+`tmt --context foo=bar --context baz=qux,quux run ...`
+
+- Allow only specific provision plugins:
+
+`tmt run -a provision --allowed-how 'container|local|virtual'`
+
+- Disable output capturing and interact directly with the test from the terminal:
+
+`tmt run --all execute --how tmt --interactive`
```

Comment:
I think these are a bit too specialized, and can depend on the implementation. How about having a tl;dr part that is more abstract demystifying what the keywords do
```suggestion
- Overwrite (or add) a step that is otherwise read from the `.fmf` definition

`tmt run -a <discover|provision|prepare|...> (--insert) ...`
```

Reply:
I've kept the interactive. Ok with you @LecrisUT ? Better suggestion?

Reply:
Yeah, maybe can add a parenthesis of why it is useful? Never tried that combination so not sure what specific to put there, but maybe what's the expectation, e.g. is it like running it line-by-line?